### PR TITLE
add state segmenting to polls page

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: templates/v2/public/polls.haml [hunk 1]
```diff
@@ -132,10 +132,22 @@
...
+              .state-dropdown.absolute.z-50(id="states-{{question.id}}" style="margin-left:-1px")
+                .border.border-gray-400.text-left.bg-white.h-64(style="overflow-y:scroll;margin-top:.5rem;margin-left:-1rem;")
+                  .pt-1.text-black
+                    {% for state in states %}
+                    .pl-2.pr-4.py-1.state-segment(data-state='{{state.id}}' data-question="{{question.id}}" class="hover:bg-gray-200 {% if forloop.counter0 < 3 %}selected{% endif %}")
+                      .checkbox.float-left
+                        %i.fa.fa-check
+                      .ml-4.whitespace-no-wrap
+                        {{ state.name }}
+                    {% endfor %}
```
Comment (norkans7) [inline templates/v2/public/polls.haml]: I think we need to keep using the haml tags way 

```
-for state in states
```
and omit the closing tags

Comment (nicpottier) [reply templates/v2/public/polls.haml]: Why do we need to?

File: templates/v2/public/polls.haml [hunk 2]
```diff
@@ -140,13 +140,12 @@
...
-                        {{ state.name }}
-                    {% endfor %}
+                    - for state in states
+                      .pl-2.pr-4.py-1.state-segment(data-state='{{state.id}}' data-question="{{question.id}}" class="hover:bg-gray-200 {% if forloop.counter0 < 3 %}selected{% endif %}")
+                        .checkbox.float-left
+                          %i.fa.fa-check
+                        .ml-4.whitespace-no-wrap
+                          {{ state.name }}
 
           {% else %}
```
Comment (norkans7) [inline templates/v2/public/polls.haml]: Same for `if/else` ?

Comment (nicpottier) [reply templates/v2/public/polls.haml]: For now lets merge, I'll change everything based on https://twist.com/a/89176/ch/222296/t/1052756/

**Pull request merged**